### DISCUSSION TITLE
Website SSL cert out of date

### DISCUSSION THREAD

**Discussion by SickProdigy** [visualboyadvance-m/visualboyadvance-m#1090] [General]
Hello, would like to inform website cert is out of date 2/5/23

**Comment by ZachBacon** [comment]
Shouldn't be. I've set up the server to auto renew the SSL cert with let's encrypt each month. But I'll take a look at it when I'm at home at the computer.

**Comment by Squall-Leonhart** [comment]
Certificate is valid till April.

**Comment by ZachBacon** [comment]
alright, refreshed the certs across the board, it should be fixed now
